ansible_connect/__main__.py

```diff
@@ -80,18 +83,17 @@ def run():
         for command in args.command:
             output[command] = conn.cliconf.get(command)
 
-        for key, value in iteritems(output):
+        for _key, value in iteritems(output):
```

Comment:
this should be refactored to `itervalues` instead of `iteritems` since `key` is not needed.  Also change the import statement from earlier